### ISSUE TITLE
Return value when tcset faild

### ISSUE THREAD

**Issue by twdkeule** [thombashi/tcconfig#71]
When tcset fails because there is already a qdisc, the return value is zero, indicating succes.
```
$ sudo tcset --device lo --rate 10M; echo $?
[NOTICE] tcconfig: failed to add qdisc: qdisc already exists (dev lo, handle=17ce:, algo=htb). execute with --overwrite option if you want to overwrite the existing rules.execute with --add option if you want to add a new rule in addition to the existing rules.
0
```

**Comment by thombashi** [comment]
Thank you for your report.

I will fix this along with #68 and #70.

**Comment by thombashi** [comment]
I've fixed the problem at `tcconfig 0.12.0`
